onboarding/src/main/kotlin/io/homeassistant/companion/android/compose/HAPreviews.kt

```diff
@@ -0,0 +1,16 @@
+package io.homeassistant.companion.android.compose
+
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+
+/**
+ * Multipreview annotation that represents various device sizes. Add this annotation to a composable
+ * to render various devices.
+ */
+@Preview(name = "phone", device = "spec:width=360dp,height=640dp", group = "phone")
+@Preview(name = "phone_landscape", device = "spec:width=411dp,height=891dp,orientation=landscape", group = "phone")
```

Comment:
Why does this one have different dp values from the portrait phone?

(I think 411dp is more average than 360dp for the small side, but am not sure. What are the Compose defaults?)